apps/vr-tests-react-components/screener.config.js

```diff
@@ -17,26 +17,33 @@ function getCurrentHash() {
 
   return '';
 }
+/**
+ *
+ * @param {string} SCREENER_API_KEY
+ * @param {string} BUILD_SOURCEBRANCHNAME
+ * @param {string} DEPLOYURL
+ * @param {string} SYSTEM_PULLREQUEST_TARGETBRANCH
+ * @returns
+ */
+function getConfig(SCREENER_API_KEY, BUILD_SOURCEBRANCHNAME, DEPLOYURL, SYSTEM_PULLREQUEST_TARGETBRANCH) {
+  const baseBranch = SYSTEM_PULLREQUEST_TARGETBRANCH
```

Comment:
does the target branch still include `ref/heads` ?

Reply:
I have yet to see another value for this env variable other than `master` but I left it in the case the target branch would be different